amenity-points.mss

```diff
@@ -1557,13 +1590,9 @@
 
   // waste_disposal tagging on ways - tagging on nodes is defined later
   [feature = 'amenity_waste_disposal'][zoom >= 19] {
-    [access = null],
-    [access = 'permissive'],
-    [access = 'yes'] {
```

Comment:
When you skip original value checking, waste disposals are rendered always. I think it should be just restored to the original state, so private waste disposals will not be rendered at all.

Reply:
oh yes I completely forgot the special rules for `waste_disposal`. Corrected for both nodes and ways.